.claude/hooks/agent-verify-gate.sh

```
@@ -0,0 +1,78 @@
+      fi
+    done
+  fi
+fi
+
+# --- biome check on changed files ---
+if [ -x "$BIOME" ]; then
+  CHANGED_ALL=$(git diff --name-only --diff-filter=ACM main...HEAD 2>/dev/null | head -50 || true)
+  if [ -n "$CHANGED_ALL" ]; then
+    BIOME_FILES=$(echo "$CHANGED_ALL" | grep -E '\.(ts|tsx|js|jsx|json)$' || true)
+    if [ -n "$BIOME_FILES" ]; then
+      if ! "$BIOME" check $BIOME_FILES 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)